crates/bevy_ecs/src/system/commands.rs

```diff
@@ -15,7 +22,9 @@ pub trait Command: Send + Sync + 'static {
 /// A queue of [`Command`]s.
 #[derive(Default)]
 pub struct CommandQueue {
-    commands: Vec<Box<dyn Command>>,
+    pub(crate) commands: Vec<Box<dyn Command>>,
+    #[cfg(feature = "command_panic_origin")]
+    pub(crate) system_name: Option<Cow<'static, str>>,
```

Comment:
Should this be `Option<_>`?
inside `fn init` `system_name` is always set as Some.

Reply:
the struct `CommandQueue` is pub and can be built using `default()` (done in `LightsNode` and in `RenderResourcesNode`). In those case a None is better than an empty string I think.

Though with the field behind a feature it's less of an issue I guess

Reply:
Could we just `impl Default for CommandQueue`?

Reply:
Unless it's actually possible for it to be `None`...

Reply:
It can be `None`!